project/api/serializers.py

```diff
@@ -137,6 +173,7 @@ class NarrativeSerializer(ModelSerializer):
 
     start_year = SerializerMethodField()
     end_year = SerializerMethodField()
+    votes = SerializerMethodField()
```

Comment:
Can we also add a counter for narrations in narrative?
It might be valuable for showing number of cards in narrative before selection.
One of the old prototypes
![image](https://user-images.githubusercontent.com/61106/59532310-9b441d00-8ee8-11e9-87e7-fe5b6398f571.png)

https://sketch.cloud/s/ZdyEa/a/mqqybk